2011-02-28  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/sparc/sparc64/multiarch/memset.S(__bzero): call
	__bzero_ultra1 instead of __memset_ultra1.

diff a/sysdeps/sparc/sparc64/multiarch/memset.S b/sysdeps/sparc/sparc64/multiarch/memset.S
--- a/sysdeps/sparc/sparc64/multiarch/memset.S
+++ b/sysdeps/sparc/sparc64/multiarch/memset.S
@@ -114,8 +114,8 @@ ENTRY(__bzero)
 	 nop
 9:
 # ifdef SHARED
-	sethi	%gdop_hix22(__memset_ultra1), %o1
-	xor	%o1, %gdop_lox10(__memset_ultra1), %o1
+	sethi	%gdop_hix22(__bzero_ultra1), %o1
+	xor	%o1, %gdop_lox10(__bzero_ultra1), %o1
 # else
 	set	__bzero_ultra1, %o1
 # endif
